### PR TITLE
[README] updated logo in readme

### LINKED ISSUE #313: [Readme] Consistently present Meshery logo
#### Current State

The logo in the readme is not consistent with the one in meshery/meshery repo make it consistent :)

#### Desired State

Also check it even in the dark mode , make sure its consistent.

---

#### [Contributor Guides](https://docs.meshery.io/project/contributing) and Resources
- 📝 [Meshery Adapters Spreadsheet](https://docs.google.com/spreadsheets/d/1rGGpSXC68iDJzNRU-qZIRQsZUwkt1qLFg7JCs4pfvbU/edit?usp=sharing)
- 📚 [Instructions for contributing to documentation](https://docs.meshery.io/project/contributing/contributing-docs)
   - Meshery documentation [site](https://docs.meshery.io/) and [source](https://github.com/meshery/meshery/tree/master/docs)
- 🛠 [Meshery Build & Release Strategy](https://docs.meshery.io/project/build-and-release)
- 🎨 Wireframes and designs for Meshery UI in [Figma](https://www.figma.com/file/SMP3zxOjZztdOLtgN4dS2W/Meshery-UI)
- 🙋🏾🙋🏼 Questions: [Layer5 Discussion Forum](https://discuss.layer5.io) and [Layer5 Community Slack](http://slack.layer5.io)

### PR DESCRIPTION
**Pull request by nishantn31**
Signed-off-by: nishantn31 <nishantn31@gmail.com>

**Description**
Updated the logo in readme and made it consistent with meshery repo readme logo
This PR fixes #313 

**Notes for Reviewers**

**[Signed commits](../CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**

- [x] Yes, I signed my commits.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,8 @@
-<p style="text-align:center;" align="center"><a href="https://layer5.io/meshery"><img align="center" style="margin-bottom:20px;" src="https://raw.githubusercontent.com/layer5io/layer5/master/.github/assets/images/meshery/meshery-logo-tag-light-text-side.png"  width="70%" /></a><br /><br /></p>
+<p style="text-align:center;" align="center"><a href="https://layer5.io/meshery"><picture align="center">
+  <source media="(prefers-color-scheme: dark)" srcset="https://raw.githubusercontent.com/meshery/meshery/master/.github/assets/images/meshery/meshery-logo-dark-text-side.svg"  width="70%" align="center" style="margin-bottom:20px;">
+  <source media="(prefers-color-scheme: light)" srcset="https://raw.githubusercontent.com/meshery/meshery/master/.github/assets/images/meshery/meshery-logo-light-text-side.svg" width="70%" align="center" style="margin-bottom:20px;">
+  <img alt="Shows an illustrated light mode meshery logo in light color mode and a dark mode meshery logo dark color mode." src="https://raw.githubusercontent.com/meshery/meshery/master/.github/assets/images/meshery/meshery-logo-tag-light-text-side.png" width="70%" align="center" style="margin-bottom:20px;">
```
Comment (leecalcote) [inline README.md:4]: Thanks! Will you change the alt text to simply “Meshery logo”?

in order to enhance maintainability cross on the repos, are you able to copy and paste directly from the other read me?


Ultimately, we want to drop the tagline “multi-mesh manager”. We are just looking for the word “Meshery” without any tagline. Will you confirm that this is the version you are using?

**Review by leecalcote — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (nishantn31) [reply README.md:4]: Yes I have copied the same image from meshery repo and it does not include the tagline "the multimesh manager"

File: README.md [hunk 2]
```diff
@@ -1,4 +1,8 @@
-<p style="text-align:center;" align="center"><a href="https://layer5.io/meshery"><img align="center" style="margin-bottom:20px;" src="https://raw.githubusercontent.com/layer5io/layer5/master/.github/assets/images/meshery/meshery-logo-tag-light-text-side.png"  width="70%" /></a><br /><br /></p>
+<p style="text-align:center;" align="center"><a href="https://layer5.io/meshery"><picture align="center">
+  <source media="(prefers-color-scheme: dark)" srcset="https://raw.githubusercontent.com/meshery/meshery/master/.github/assets/images/meshery/meshery-logo-dark-text-side.svg"  width="70%" align="center" style="margin-bottom:20px;">
+  <source media="(prefers-color-scheme: light)" srcset="https://raw.githubusercontent.com/meshery/meshery/master/.github/assets/images/meshery/meshery-logo-light-text-side.svg" width="70%" align="center" style="margin-bottom:20px;">
```
Comment (legendarykamal) [inline README.md:3]: Needs modification , light mode doesn't look appropriate .

image

File: README.md [hunk 3]
```diff
@@ -1,4 +1,8 @@
...
+  <img alt="Meshery logo" src="https://raw.githubusercontent.com/meshery/meshery/master/.github/assets/images/meshery/meshery-logo-tag-light-text-side.png" width="70%" align="center" style="margin-bottom:20px;">
```
Comment (legendarykamal) [inline README.md:4]: add an alt to the img.

**Review by legendarykamal — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (nishantn31) [reply README.md:3]: @legendarykamal It is working fine for me both on light and dark mode. Can you share more details - platform/browser etc
 
image

File: README.md (hunk 3 above)
Comment (nishantn31) [reply README.md:4]: I have already added the alt text in img as "Meshery logo"

File: README.md (hunk 2 above)
Comment (leecalcote) [reply README.md:3]: @legendarykamal

**Comment by leecalcote** [comment]
Yay!

**Comment by leecalcote** [comment]
More help needed with this second Meshery logo, @nishantn31 - 
9F0CB2A9-9224-454B-BED5-C9B98EC1526E

File: README.md (hunk 2 above)
Comment (legendarykamal) [reply README.md:3]: ok @nishantn31  must have been a glitch from my end, rechecked looks fine ,
congrats your first PR got merged 🥳 .

File: README.md (hunk 3 above)
Comment (legendarykamal) [reply README.md:4]: ok so the alt was beffore the src 👍

**Pull request merged**